File: androidPdfReader/app/src/main/java/com/example/taher144/pdfreaderlite/data/model/PdfBookmark.kt
package com.example.taher144.pdfreaderlite.data.model

/**
 * Single in-document reading bookmark. Coordinates are in PDF page space
 * (origin at top-left of the page), matching [androidx.pdf.PdfPoint].
 */
data class PdfBookmark(
    val documentId: String,
    val pageIndex: Int,
    val x: Float,
    val y: Float,
)

File: PDFReaderT/PDFReaderT/DataModel/PdfBookmark.swift
//
//  PdfBookmark.swift
//  PDFReaderT
//

import Foundation

/// Single in-document reading bookmark. Coordinates are in PDF page space
/// (origin at bottom-left of the page), matching PDFKit.
struct PdfBookmark: Equatable {
    let documentId: String
    let pageIndex: Int
    let x: Double
    let y: Double
}
